### PR TITLE
WRAN-7-automatic-lint

### PR DESCRIPTION
**Pull request by keenangraham**
Automatic linting of files that still need it.

### CODE REVIEW

File: ENCODE_fastq_trimmer.py [hunk 1]
```diff
@@ -79,7 +79,8 @@ def trim_file(url, connection, filename, size):
                 header = next(gzfile)
```
Comment (hitz) [inline ENCODE_fastq_trimmer.py:79]: There are import errors in this file, L95-96

**Review by hitz — CHANGES_REQUESTED** [review]
This is kind of tricky to review since all you did was reformat.  Did you check to see if all the scripts still run?
I basically just looked at the python files in sublime and looked for yellow and red marks.

File: ENCODE_fastq_trimmer.py (hunk 1 above)
Comment (keenangraham) [reply ENCODE_fastq_trimmer.py:79]: That's another issue, as it doesn't work on master either:
```
DN0a22f83f:pyencoded-tools keenan$ python ENCODE_fastq_trimmer.py 
Traceback (most recent call last):
  File "ENCODE_fastq_trimmer.py", line 167, in <module>
    main()
  File "ENCODE_fastq_trimmer.py", line 94, in main
    key = encodedcc.ENC_Key(args.keyfile, args.key)
NameError: name 'encodedcc' is not defined
```
The real question is who commented out encodedcc from this file in the first place and why?
screen shot 2017-10-19 at 3 26 17 pm

**Comment by keenangraham** [comment]
Fair question but the purpose of this ticket isn't to see if the scripts work. It's to prevent an initial automatic formatting commit for each new file I touch (so the diff isn't confused by 100 formatting changes when I go to actually fix the script). Should we really be concerned that the automatic formatting might break something that wasn't already broken?

**Comment by hitz** [comment]
The issue is simply that you might have introduced a typo.

**Pull request merged**